resources/themes/STARTERTHEME/Gruntfile.js

```diff
@@ -13,10 +13,13 @@ module.exports = function (grunt) {
         files: [{
           expand: true,
           cwd: 'sass',
-          src: ['*.scss', '**/*.scss'],
+          src: '**/*.scss',
```

Comment:
You don't need the "sass/" part of the path since it was already defined in "cwd" :)